ext/js/display/display-anki.js

```diff
@@ -370,6 +370,31 @@ export class DisplayAnki {
         }
     }
 
+    /**
+     * @param {HTMLButtonElement} button
+     */
+    _showDuplicateAddButton(button) {
+        const isKanjiAdd = button.dataset.mode === 'term-kanji';
+
+        const title = button.getAttribute('title');
+
```

Comment:
This is a minor stylistic nitpick, and it's hard to perfectly describe what I consider to be "good" IMO, but I prefer to avoid having lots of blank newlines between related parts of the code.

Either one of these I think is fine:
```js
    _showDuplicateAddButton(button) {
        const isKanjiAdd = button.dataset.mode === 'term-kanji';
        const title = button.getAttribute('title');
        const {hotkey} = button.dataset;
        const actionIcon = button.querySelector('.action-icon');

        if (title) {
            button.setAttribute('title', title.replace(/Add (?!duplicate)/, 'Add duplicate '));
        }
        if (hotkey) {
            button.dataset.hotkey = hotkey.replace(/Add (?!duplicate)/, 'Add duplicate ');
        }
        if (actionIcon instanceof HTMLElement) {
            actionIcon.dataset.icon = isKanjiAdd ? 'add-duplicate-term-kanji' : 'add-duplicate-term-kana';
        }
    }
```

```js
    _showDuplicateAddButton(button) {
        const isKanjiAdd = button.dataset.mode === 'term-kanji';

        const title = button.getAttribute('title');
        if (title) {
            button.setAttribute('title', title.replace(/Add (?!duplicate)/, 'Add duplicate '));
        }

        const {hotkey} = button.dataset;
        if (hotkey) {
            button.dataset.hotkey = hotkey.replace(/Add (?!duplicate)/, 'Add duplicate ');
        }

        const actionIcon = button.querySelector('.action-icon');
        if (actionIcon instanceof HTMLElement) {
            actionIcon.dataset.icon = isKanjiAdd ? 'add-duplicate-term-kanji' : 'add-duplicate-term-kana';
        }
    }
```

Or just entirely removing all the empty lines, since the functions are small enough that the compactness shouldn't harm understandability. This comment I think may also be related to some of the changes in backend.js, but this area stood out the most.